RecoLocalCalo/HcalRecAlgos/src/parseHFPhase1AlgoDescription.cc

```diff
@@ -30,6 +30,12 @@ parseHFPhase1AlgoDescription(const edm::ParameterSet& ps)
         const bool rejectAllFailures =
             ps.getParameter<bool>("rejectAllFailures");
 
+        float minChargeForUndershoot = -10000.f, minChargeForOvershoot = -10000.f;
+        if (ps.existsAs<double>("minChargeForUndershoot"))
```

Comment:
since this is for the future implementations, the "existsAs" can not be in the supported development plans.
It looks like it's time to revisit the design to comply with the configuration validation
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp
this can help

For the current PR, simple addition of the parameters to the available fillDescriptions should apparently work.